src/DependencyGraph/DependencyGraph.ts

```diff
@@ -792,16 +731,13 @@ export class DependencyGraph {
   private formulaDependencyQuery: (vertex: Vertex) => Maybe<[SimpleCellAddress, CellDependency[]]> = (vertex: Vertex) => {
     let formula: Ast
     let address: SimpleCellAddress
-    if (vertex instanceof FormulaCellVertex) {
+    if (vertex instanceof FormulaCellVertex || vertex instanceof MatrixVertex) {
```

Comment:
```suggestion
    if (vertex instanceof FormulaVertex) {
```

could it work like that?